matrix-sdk-android/src/main/java/org/matrix/android/sdk/api/Matrix.kt

```diff
@@ -29,6 +29,7 @@ import org.matrix.android.sdk.api.legacy.LegacySessionImporter
 import org.matrix.android.sdk.api.network.ApiInterceptorListener
 import org.matrix.android.sdk.api.network.ApiPath
 import org.matrix.android.sdk.api.raw.RawService
+import org.matrix.android.sdk.api.session.securestorage.SecureStorageService
```

Comment:
Should we keep the `SecureStorageService` inside the `api.session` package?

Reply:
I also agree to that `sdk.api.securestorage` seems better.